src/lib/viewers/doc/DocControls.tsx

```diff
@@ -37,6 +42,8 @@ export default function DocControls({
     pageNumber,
     scale,
 }: Props): JSX.Element {
+    const colors = Object.values(AnnotationColor);
```

Comment:
Does this need to be redefined every render?

Reply:
Nope, will move above this function.